features/step_definitions/lr_casework_search_for_deed.rb

```diff
@@ -6,16 +6,17 @@
   @response = HTTP.get(full_url)
 end
 
-Then(/^I verify the returned deed information$/) do
+Then(/^I verify the returned deed information(?: is "([^"]*)")?$/) do |deed_sts|
   data = JSON.parse(@response)
+  deed_sts ||= 'DRAFT'
   if data.length > 0
     deed_count = 0
     # Checks each returned deed for key values: status, token. And value DRAFT.
     while deed_count <= (data.length - 1)
-      status = data[deed_count].include?('status')
-      token = data[deed_count].include?('token')
-      draft = data[deed_count]['status'].include?('DRAFT')
-      tok_val = data[deed_count]['token'].nil?
+      status = data[deed_count.to_i].include?('status')
+      token = data[deed_count.to_i].include?('token')
+      draft = data[deed_count.to_i]['status'].include?(deed_sts)
```

Comment:
Would be tempted to rename this from draft to soemthing more meaningful now status is a variable thing, something like status_val or whatever you think is more suitable.